content/en/docs/tasks/observability/metrics/using-istio-dashboard/index.md

```diff
@@ -107,7 +107,7 @@ the example application throughout this task.
 ### About the Grafana add-on
```

Comment:
nit if you need to make more changes: add-on is  mispelled. We should be using add-on per the contribution docs.

Reply:
If more changes are required by this PR, I can do that. If not, perhaps in a follow-on.

I am not anticipating any further changes at this point, however ;).

Reply:
Well, spoke too soon. Had more changes. Fixed `add-on` -> `addon` here.